samples/samples-csharp/host.json

```diff
@@ -8,7 +8,7 @@
             }
         },
         "logLevel": {
-          "default": "Information"
+          "default": "Debug"
```

Comment:
Now that we use debug logging for more stuff I had to change this for tests. We originally had it as Information so customers didn't copy the debug settings, but in this case I think it's fine to have that vs. adding some functionality to switch the log level at runtime (which I had looked into earlier and didn't see an obvious way to do)